### PR TITLE
Validate DnD action for FluidSlot on server

### PR DESCRIPTION
**Pull request by miozune**

### CODE REVIEW

File: src/main/java/com/cleanroommc/modularui/value/sync/FluidSlotSyncHandler.java [hunk 1]
```diff
@@ -103,7 +103,9 @@ public void readOnClient(int id, PacketBuffer buf) {
     @Override
     public void readOnServer(int id, PacketBuffer buf) {
         if (id == 0) {
-            read(buf);
+            if (this.phantom) {
+                read(buf);
```
Comment (brachy84) [inline src/main/java/com/cleanroommc/modularui/value/sync/FluidSlotSyncHandler.java:107]: Why is this needed?

File: src/main/java/com/cleanroommc/modularui/screen/Tooltip.java [hunk 2]
```diff
@@ -81,7 +81,7 @@ public void draw(GuiContext context, @Nullable ItemStack stack) {
         int mouseX = context.getAbsMouseX(), mouseY = context.getAbsMouseY();
         IconRenderer renderer = IconRenderer.SHARED;
         List<String> textLines = lines.stream().filter(drawable -> drawable instanceof IKey).map(key -> ((IKey) key).get()).collect(Collectors.toList());
-        RenderTooltipEvent.Pre event = new RenderTooltipEvent.Pre(stack, textLines, mouseX, mouseY, screen.width, screen.height, thimaxWidth, TextRenderer.getFontRenderer());
+        RenderTooltipEvent.Pre event = new RenderTooltipEvent.Pre(stack, textLines, mouseX, mouseY, screen.width, screen.height, maxWidth, TextRenderer.getFontRenderer());
```
Comment (brachy84) [inline src/main/java/com/cleanroommc/modularui/screen/Tooltip.java:84]: that was supposed to be `this.maxWidth`, but had incredible skill issue during merge

File: src/main/java/com/cleanroommc/modularui/value/sync/FluidSlotSyncHandler.java (hunk 1 above)
Comment (miozune) [reply src/main/java/com/cleanroommc/modularui/value/sync/FluidSlotSyncHandler.java:107]: Explained in the DM on Discord

**Review by brachy84 — APPROVED** [review]
